protos/telemetry/telemetry.proto

```diff
@@ -757,6 +757,9 @@ message FixedwingMetrics {
     float airspeed_m_s = 1 [(mavsdk.options.default_value)="NaN"]; // Current indicated airspeed (IAS) in metres per second
     float throttle_percentage = 2 [(mavsdk.options.default_value)="NaN"]; // Current throttle setting (0 to 100)
     float climb_rate_m_s = 3 [(mavsdk.options.default_value)="NaN"]; // Current climb rate in metres per second
+    float groundspeed_m_s = 4 [(mavsdk.options.default_value)="NaN"]; // Current groundspeed metres per second
+    float heading_deg = 5 [(mavsdk.options.default_value)="NaN"]; // Current heading in compass units (0-360, 0=north)
+    float altitude_msl = 6 [(mavsdk.options.default_value)="NaN"]; // Current altitude in metres (MSL)
```

Comment:
Maybe this would be more consistent with the others?

```suggestion
    float absolute_altitude_m = 6 [(mavsdk.options.default_value)="NaN"]; // Current altitude in metres (MSL)
```

Reply:
Yep no problem, will do

- [ ] absolute_altitude_m

Reply:
Done. Sorry had to test on two different branches, which is why I didn't commit directly. Should be there though.